add function to check if is implicit

ruixiangcui opened the pull request:
I also added three files with implicit arguments from EWT to test _files.

ruixiangcui committed:
check if is implicit in candidate

ruixiangcui committed:
check if is implicit in candidate

ruixiangcui committed:
check if is implicit in candidate

ruixiangcui committed:
check if is implicit in candidate

danielhers commented on ucca/constructions.py:
```diff
@@ -70,7 +71,8 @@ def __init__(self, edge, reference=None, reference_yield_tags=None, verbose=Fals
         self.terminals = self.edge.child.get_terminals()
         self._terminal_yield = positions(self.terminals)
-        self._terminal_yield_no_punct = positions(self.edge.child.get_terminals(punct=False))
+        self._terminal_yield_no_punct = positions(self.edge.parent.get_terminals(punct=False)) if self.is_implicit\
```
Good. Just had an idea: it can be rewritten `positions((self.edge.parent if self.is_implicit else self.edge.child).get_terminals(punct=False))`

danielhers commented on ucca/constructions.py:
```diff
@@ -179,7 +184,7 @@ def __str__(self):
     Construction("expletives", "Expletives",
                  lambda c: c.tokens <= {"it", "there"} and EdgeTags.Function in c.edge.tags),
-    Categories(),
+    Categories()
```
What about adding an "implicit" construction? I thought you added that.

danielhers commented on ucca/layer0.py:
```diff
@@ -7,7 +7,6 @@
 :class:`core`.Node, and can have one of two tags: Word or Punctuation.
 
 """
-
```
Avoid changes to files if they're only whitespace

danielhers commented:
Needs fixing: expected results in tests (ucca/tests/test_evaluation.py): https://travis-ci.org/danielhers/ucca/jobs/615591438
Implicit units used to be excluded from evaluation, but now they're included in the primary score. The expected scores should therefore be updated to account for that.

danielhers commented:
Also, expected results for evaluation should be added for the three new files (`implicit1.xml`, `implicit2.xml`, `implicit3.xml`) in https://github.com/danielhers/ucca/blob/master/ucca/tests/test_evaluation.py#L310

ruixiangcui committed:
check if is implicit in candidate

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
modify implicit values in test_constructions

ruixiangcui committed:
add two implicit files

ruixiangcui commented:
Sorry that I committed so many times. Now I have generally commanded how to use Git. 
This time I added implicit edges into Constructions; Added two test files with implicit arguments; modified test parameters (changed values and added implicit features) for  passage1-passage2, simple1-simple2, function1-function2, 120_parsed-standard3, and implicit1-implicit1_ref, implicit2-implicit2. 
Thanks for checking them!
